try punting emails to celery queue with no workers

nickpell opened the pull request.

Commit:
try punting emails to celery queue with no workers

dannyroberts reviewed (commented):
Is the idea that we will keep this indefinitely, or that we not merge this and just deploy from this branch?

nickpell commented:
I might suggest deploying this branch since it forks the current branch running on production, so we don't have to deal with any other changes since last deploy at this moment.

If it works, I'd say then merge this to master, so if anyone deploys master they won't bring down prod.

dannyroberts commented:
What do we have to do not to mess up other envs, if we merge this to master?

Commit:
only affect prod/staging

nickpell commented:
Updated to only affect staging (for testing) and prod

emord commented on corehq/ex-submodules/dimagi/utils/django/email.py:
```diff
@@ -26,6 +27,29 @@
+
+@task(queue="email_queue_punt",
+      bind=True, default_retry_delay=15 * 60, max_retries=10, acks_late=True)
+def _send_HTML_email_punt(subject, recipient, html_content, text_content=None,
```
When you have `bind=True` I think tha the first argument should be `self`

nickpell replied:
updated

Commit:
self

Commit:
put other email sending tasks on email_queue_punt
